lib/node_modules/@stdlib/stats/base/cuminabs/docs/types/test.ts

```diff
@@ -27,6 +28,7 @@ import cuminabs = require( './index' );
 	const y = new Float64Array( 10 );
 
 	cuminabs( x.length, x, 1, y, 1 ); // $ExpectType NumericArray
```

Comment:
```suggestion
	cuminabs( x.length, x, 1, y, 1 ); // $ExpectType Float64Array
```

Based on the type defns, the output array type should be preserved.